src/codegen/codegen.ts

```diff
@@ -22,30 +23,72 @@ export function generateScript({
   generator,
 }: GenerateScriptParams): string {
   return `
-    ${generateScriptHeader()}
-    ${REQUIRED_IMPORTS.map(generateImportStatement).join('\n')}
+    ${generateScriptHeaderComment()}
+    ${generateImports(generator)}
 
     export const options = ${generateOptions(generator.options)}
 
     ${generateVariableDeclarations(generator.testData.variables)}
+    ${generateDataFileDeclarations(generator.testData.files)}
 
     export default function() {
       ${generateVUCode(recording, generator.rules, generator.options.thinkTime)}
     }
   `
 }
 
+export function generateImports(generator: GeneratorFileData): string {
+  const hasDataFiles = generator.testData.files.length > 1
+  const hasCsvDataFiles = generator.testData.files.some(({ name }) =>
+    name.toLowerCase().endsWith('csv')
+  )
```

Comment:
I'm not sure if  inferring the file type from the filename is a good solution. We can probably store the type along the name. Although it won't change the fact the type is inferred in a similar way elsewhere